ramls/acquisitions-units.raml

```diff
@@ -54,3 +56,31 @@ resourceTypes:
       put:
         description: Update acquisitions unit
         is: [validate]
+  /memberships:
+    type:
+      collection:
+        exampleCollection: !include acq-models/mod-orders-storage/examples/acquisitions_unit_membership_collection.sample
+        exampleItem: !include acq-models/mod-orders-storage/examples/acquisitions_unit_membership_post.sample
+        schemaCollection: acquisitions-unit-membership-collection
+        schemaItem: acquisitions-unit-membership
+    post:
+      description: Create new acquisitions units membership
+      is: [validate]
+    get:
+      description: Get list of acquisitions units memberships
+      is: [
+        searchable: {description: "with valid searchable fields: for example protectRead", example: "[\"protectRead\", \"false\"]"},
+        pageable
+      ]
+    /{id}:
+      uriParameters:
+        id:
+          description: The UUID of an acquisitions units membership
+          type: UUID
+      type:
+        collection-item:
+          exampleItem: !include acq-models/mod-orders-storage/examples/acquisitions_unit_membership_post.sample
```

Comment:
Please use `acquisitions_unit_membership_get.sample` here.
Also if you can, please replace `acquisitions_unit_post.sample` by `acquisitions_unit_get.sample` for `/acquisitions-units/units/{id}` just to highlight that `id` will be part of response body...